src/web/wms/src/main/java/org/geoserver/wms/web/data/AbstractStylePage.java

```diff
@@ -59,6 +87,130 @@
  */
 @SuppressWarnings("serial")
 public abstract class AbstractStylePage extends GeoServerSecuredPage {
+    
+    private static String[] IMAGE_EXTENSIONS = new String[] {"png", "jpg", "jpeg", "gif", "svg"};
```

Comment:
I wonder if this should be configurable, in case there are extensions that add supported image types (I don't know of any currently, but MBStyle may eventually support pbf images).
Probably fine as-is, and if anyone adds a new image format they can be responsible for making this support it

Reply:
Agreed, unless you have a different suggestion of where to configure it in Spring perhaps